src/LetterChanger.java

```diff
@@ -1,57 +1,52 @@
+import java.util.List;
 import java.util.Scanner;
 
 class LetterChanger {
     private static Scanner scanner = new Scanner(System.in);
-
-    public static void main(String[] args) {
-        int numberOfWords = getNumberOfWords();
-        String[] words = readWords(numberOfWords);
-        String change = readChangeString();
-        if (change == "upper") {
-            changeWordsToUpperCase(words);
-        } else if(change == "lower") {
-            changeWordsToLowerCase(words);
-        } else {
-            System.out.println("Nieprawidłowy tekst, zostawiam oryginalny tekst");
+    OperationsOnWords onWords = new OperationsOnWords();
```

Comment:
I wouldn't create new class just to store words list and then transform them to upper or lower case. I would do everything in this class.

Reply:
everything you mean method which change to upper or lower case also ? I wanted "take" some method to other class according to "good practices". BTW remove list from that class and define new one in this one. Look and tell what do you think.

Reply:
Yeah, it is a simple task to do upper or lower. If you want to have it in different class then create utility class with static methods to change text to upper or lower case

Reply:
Ok I made changes.